tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -5197,4 +5221,38 @@ void ASMain()
   ValidateStructMember(2, L"bigOne", 24*8);
 }
 
+TEST_F(PixTest, DxilPIXDXRInvocationsLog_SanityTest) {
+
+    const char* source = R"x(
+struct MyPayload
+{
```

Comment:
Is it possible to create FileCheck test instead of unit test?

Reply:
Yes. Are we concerned about the running time for unit tests or something?

Reply:
(Meaning, is it OK to have FileCheck tests AND these unit tests? Or do you mean it's better to have just filecheck tests?)

Reply:
See:

https://github.com/microsoft/DirectXShaderCompiler/pull/5322

Reply:
If the FileCheck and unit test are testing same thing, it is better to have just filecheck tests.

Reply:
I've added a file-check test